api/src/main/java/io/minio/HttpResponse.java

```diff
@@ -52,6 +52,13 @@ public ResponseBody body() {
     return this.response.body();
   }
 
+  /**
+   * closes the response.
+   */
+  public void close() {
```

Comment:
Keep this clean up as seprate PR.

Reply:
this is also needed otherwise we had bugs where the body is not closed properly.

Reply:
Where is the code the body wasn't closed?

Reply:
Error was shown by okhttp that we are leaking response and it is recommended that we close response.close() instead of response.body().close()